iot-hub/Tutorials/Routing/SimulatedDevice/Program.cs

```diff
@@ -25,22 +26,31 @@
 
 namespace SimulatedDevice
 {
-    class Program
+    internal class Program
     {
         private static DeviceClient s_deviceClient;
-        private readonly static string s_myDeviceId = "Contoso-Test-Device";
-        private readonly static string s_iotHubUri = "<iot-hub-name-goes-here>.azure-devices.net";
+        private static string s_myDeviceId = "Contoso-Test-Device";
```

Comment:
This can be removed and just grabbed from params, right?

If we want a default, set it in params.